controller/gen/public/api.pb.go

```diff
@@ -206,9 +206,7 @@ func (m *MetricValue) String() string            { return proto.CompactTextStrin
 func (*MetricValue) ProtoMessage()               {}
 func (*MetricValue) Descriptor() ([]byte, []int) { return fileDescriptor0, []int{2} }
 
-type isMetricValue_Value interface {
```

Comment:
a lot of these pb changes are just whitespace. just want to confirm this is expected / an artifact of a different protoc version?

Reply:
Hmm, yeah, this is a good point. As far as I can tell, the previously generated bindings weren't using the latest protoc-gen-go lib. If I run `protoc-go.sh` on master, I see the same changes as the ones in this branch:

```
kl ~/workspace/conduit (master) $ ./bin/protoc-go.sh && git diff --stat
 controller/gen/common/common.pb.go                  | 24 ++++++------------------
 controller/gen/controller/telemetry/telemetry.pb.go | 13 +++++++++----
 controller/gen/proxy/destination/destination.pb.go  |  4 +---
 controller/gen/proxy/tap/tap.pb.go                  | 16 ++++------------
 controller/gen/proxy/telemetry/telemetry.pb.go      |  4 +---
 controller/gen/public/api.pb.go                     |  8 ++------
 6 files changed, 23 insertions(+), 46 deletions(-)
```

What happens if you run that script on master?

Reply:
yup, this isn't specific to your branch. some of these files have been in this state for months, also our versioning of `github.com/golang/protobuf/protoc-gen-go` in `Gopkg.lock` hasn't changed either. i suppose we should merge it.

```bash
(master)$ git diff --stat
 controller/gen/common/common.pb.go                  | 24 ++++++------------------
 controller/gen/controller/telemetry/telemetry.pb.go | 13 +++++++++----
 controller/gen/proxy/destination/destination.pb.go  |  4 +---
 controller/gen/proxy/tap/tap.pb.go                  | 16 ++++------------
 controller/gen/proxy/telemetry/telemetry.pb.go      |  4 +---
 controller/gen/public/api.pb.go                     |  8 ++------
 6 files changed, 23 insertions(+), 46 deletions(-)
```